third_party/sl4a/src/main/java/com/google/android/mobly/snippet/rpc/MethodDescriptor.java

```diff
@@ -79,10 +77,8 @@ public String toString() {
    * @throws Throwable
    */
   public Object invoke(SnippetManager manager, final JSONArray parameters) throws Throwable {
-
     final Type[] parameterTypes = getGenericParameterTypes();
     final Object[] args = new Object[parameterTypes.length];
-    final Annotation annotations[][] = getParameterAnnotations();
```

Comment:
@RpcParameter was rarely used in sl4a and is very verbose and annoying to include.
I'd much prefer to slim down what's needed to make a snippet to just the basics.
If the RPC description and names of variables are not sufficient, people can go look at the code.
If this proves to be insufficient we can always include the javadoc as a resource in the apk as a build step and dump it to the console.